addons/mail/static/src/components/composer/composer.js

```diff
@@ -207,7 +207,7 @@ class Composer extends Component {
             }
             return;
         }
-        await this.composer.postMessage();
+        await this.composer.queueMessage();
         // TODO: we might need to remove trigger and use the store to wait for the post rpc to be done
         // task-2252858
         this.trigger('o-message-posted');
```

Comment:
This change is making the following trigger not correct. (`queueMessage` is not `async` so the `await` is useless).

The trigger should be done when the queued message is actually posted (or when all of them are posted would be fine too). This will require generating and returning a deferred that is resolved at the appropriate time.